src/lib/routes/dispatcher.js

```diff
@@ -1,18 +1,33 @@
-import permissionValidator from '../validators';
+import { permissionsValidator, requestValidator } from '../validators';
+import setRequestData from './setRequestData';
 
 const dispatcher = (controller, controllerMethod) => async (
   request,
   response,
   next
 ) => {
   try {
-    const { permissionRules } = controller;
+    const { permissionRules, validationRules } = controller;
+
+    // Get all request data and setup it in data param of request
+    setRequestData(request);
```

Comment:
IMPORTANT

It should be a pure function returning the data from the request.

Always try to avoid mutation of the parameters inside a function.
Now I'm reading the code and don't understand what does it do with data, but if I do explicitly 
`request.data = getRequestDate(request)`
I can understand that now in request.data will be some kind of data from the request, and if I need to know more, I can check out the function.

Does that make sense?